tests/test_target.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+from target import Target, TargetAttr
+
+class TestTarget(unittest.TestCase):
+    def test_constructor(self):
+        target = Target()
+        self.assertIsNotNone(target)
+        self.assertGreater(target.points, 0)
+
+if __name__ == '__main__':
+    unittest.main()
```

Comment:
This unit test makes sense to me! looks like the target is being generated and the points are being assigned

Reply:
Looks good to me too! Btw I had removed the if __name__ == '__main__' lines since they don't seem to make any difference, I think Daniel mentioned that too. But definitely not a big deal

Reply:
Yeah, I realized that too when updating the test folder.  Thank you